thoth/storages/data/alembic/versions/7f7411e247d6_migrate_license_solver_data.py

```diff
@@ -0,0 +1,30 @@
+"""Migrate license solver data
+
+Revision ID: 7f7411e247d6
+Revises: 2a4632197512
+Create Date: 2022-09-19 12:26:03.463149+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "7f7411e247d6"
+down_revision = "2a4632197512"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint(
+        None, "python_package_license", ["license_name", "license_identifier", "license_version"]
+    )
+    # ### end Alembic commands ###
```

Comment:
I have some concerns over this.
I don't think create_unique_constraint automatically find and eliminates duplicates; so the autogenerated migration will not be good enough, we'd need to:
1. Find first ocurrence for each case
2. Update foreign key in other table to refer to first ocurrence
3. Remove duplicates.
Otherwise we're going to run into an IntegrityError, I think.